spec.html

```diff
@@ -3147,7 +3147,7 @@ <h2>[[GetOwnProperty]] ( _P_ )</h2>
             The normal return type is either Property Descriptor or Undefined.
           </li>
           <li>
-            If the Type of the return value is Property Descriptor, the return value must be a fully populated Property Descriptor.
+            If the return value is a Property Descriptor, the return value must be a fully populated Property Descriptor.
```

Comment:
```suggestion
            If the return value is a Property Descriptor, it must be a fully populated Property Descriptor.
```
(reads a bit better)